### PR TITLE
docker/ofa: move PARAMDIR from ~/.jsk_vil_apis to ~/.ros/data/jsk_per…

### PR DESCRIPTION
**Pull request by k-okada**
…ception/vil_params as other jsk_perception tools

### CODE REVIEW

**Comment by k-okada** [comment]
@mqcmd196 added clip classification task

File: jsk_perception/cfg/Classification.cfg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+PACKAGE = "jsk_vil_msgs"
```
Comment (mqcmd196) [inline jsk_perception/cfg/Classification.cfg:2]: PACKAGE name should be "jsk_perception" ?

File: jsk_perception/node_scripts/classification_client_node.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+class ClipClientNode:
+    def __init__(self):
+        # inference server configuration
+        self.host = rospy.get_param("~host", default="localhost")
+        self.port = rospy.get_param("~port", default=8888)
+        # clip task
+        self.clip_image_sub = rospy.Subscriber("~image", Image,
+                                                  callback=self.clip_topic_cb, queue_size=1, buff_size=2**26)
+        self.clip_result_pub = rospy.Publisher("~clip_result", ClassificationResult, queue_size=1)
```
Comment (mqcmd196) [inline jsk_perception/node_scripts/classification_client_node.py:37]: It seems these topic names should be `classification_result` or something...?

**Review by mqcmd196 — CHANGES_REQUESTED** [review]
Thank you for your feature. I added some comment about variable, config name

**Pull request merged**